### PR TITLE
Fix/p10x credit rate

### PR DESCRIPTION
**Pull request by PaulRouss3l**
For p10x we didn't had the TAEG/credit cost stuf, now we do :) 

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,6 @@ metadata/
 **/dw.js
 **/dw.json
 .env
+
+# avoid publishing translation config
+crowdin.yml
```
Comment (syjust) [inline .gitignore:26]: ok for me but, how do we share crowdin config now ?

File: cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js [hunk 2]
```diff
@@ -50,6 +50,20 @@ function getPaymentCoreInfo(plan) {
     };
 }
 
+/**
+ * Returns the amount and rate of a credit properties for a given plan
+ * @param  {Object} plan any alma plan
+ * @returns {Object} with the properties translated
+ */
+function getCreditInfo(plan) {
+    var costOfCredit = parseInt(plan.customer_total_cost_amount / 100, 10).toString() + '.' + (plan.customer_total_cost_amount % 100) + '€';
```
Comment (syjust) [inline cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js:59]: maybe use `Math.round()` when you ` / 100` ... no ?

Comment (syjust) [inline cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js:59]: it is the merchant responsibility to not load Alma if currency is not euro ?

File: .gitignore (hunk 1 above)
Comment (PaulRouss3l) [reply .gitignore:26]: I keep this file in the dev repository too, so we just have to copy it on set up/ when needed.

File: cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js (hunk 2 above)
Comment (PaulRouss3l) [reply cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js:59]: It's either `Math.round()` or `parseInt()`. round is more explicit to dev, parseInt is faster in execution.
I don't have a strong opinion on either one, if you prefer one over the other, tell me, I'll change it :)

Comment (PaulRouss3l) [reply cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js:59]: I should use `formatCurrency` which is a SFCC function to do that, you're right :), will fix

But if our merchant is not using EUR, our psp config, inside SFCC will not allow alma to be displayed.
If a merchant edit this config, yes, Alma won't work properly.

**Review by syjust — APPROVED** [review]

File: cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js (hunk 2 above)
Comment (syjust) [reply cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js]: `Math.round()` is used everywhere else.
I prefer keep this more explicit method to to the job ;)

**Pull request merged**